gke-integration-job/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
```

Comment:
IMO we should use prow/images directory

Reply:
but I doubt if we really need to create image: see changes proposed by Suleyman: https://github.com/kyma-project/test-infra/pull/98/files#diff-4, he gets rid of creating image and use bootrstap with mounted test-infra repo.